app/models/solidus_subscriptions/subscription_event.rb

```diff
@@ -9,7 +9,7 @@ class SubscriptionEvent < ApplicationRecord
     end
 
     def self.ransackable_attributes(_auth_object = nil)
-      %w[created_at details event_type id id_value subscription_id updated_at]
```

Comment:
Do `details` include useful information that should be searchable?

Reply:
In the subscription events page, no search filter available. For rendering the page, it needed the essential attributes.